app/views/layouts/application.html.erb

```diff
@@ -7,17 +7,17 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  
-  <nav class="top-bar">
-    <h1><a href="/">RubyZoo</a></h1>
 
-    <section class="top-bar-section">
-      <ul class="right">
+  <nav class="top-bar header">
+    <h1 id="site-title"><a href="/">RubyZoo</a></h1>
+
+    <section class="top-bar-section header">
+      <ul class="right enter-buttons">
         <%- if current_user -%>
-          <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+          <li><%= link_to '<button type="button">Sign Out</button>'.html_safe, destroy_user_session_path, method: :delete %>></li>
         <%- else -%>
-          <li><%= link_to 'Sign Up', new_user_registration_path %></li>
-          <li><%= link_to 'Sign In', new_user_session_path %></li>
+          <li><%= link_to '<button type="button">Sign Up</button>'.html_safe, new_user_registration_path %></li>
+          <li><%= link_to '<button type="button">Sign In</button>'.html_safe, new_user_session_path %></li>
```

Comment:
I could be wrong, but this will likely create a dangerously nested set of html. We do not want html elements nested in anchor tags as a rule. 

Alternatives: use Rails's "button_to"
OR better yet: just make it a link_to and style it to look like a button (a pattern used by a lot of folks)

Reply:
I would love to talk to you about this!